io.github.chkr1011.mqttMultimeter.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Name=mqttMultimeter
+Icon=io.github.chkr1011.mqttMultimeter
+Exec=/app/bin/mqttMultimeter
```

Comment:
never a path. should just be the executable name (and maybe substitution for command line arguments if needed)